File: Traveling/src/main/java/Project/Traveling/Model/UserDetails.java
package Project.Traveling.Model;

import jakarta.persistence.*;
import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder

public class UserDetails {
    private long userId;
    private String userName;
    private String email;
    private String password;
    private UserType userType;

}

File: Client/lakeSide-hotel/src/components/Models/UserDetails.ts
export class UserDetailsModel{
    public userName:string;
    public firstName?:string;
    public lastName?:string;
    public email:string;
    public password:string;
    public id?:string;
    public userType:string;
    constructor(userName:string,email:string,password:string,id = undefined,userType:string,firstName = undefined,lastName = undefined ){
       this.userName = userName;
       this.email = email;
       this.password = password;
       this.userType = userType;
       this.id = id;
       this.firstName = firstName;
       this.lastName = lastName;
}}